psi4/driver/driver_nbody.py

```diff
@@ -467,19 +471,22 @@ def nbody_gufunc(func, method_string, **kwargs):
     else:
         ret_ptype = ret_energy
 
-    # Build and set a wavefunction
+    # Build wfn and bind variables
     wfn = core.Wavefunction.build(molecule, 'def2-svp')
-    wfn.nbody_energy = energies_dict
-    wfn.nbody_ptype = ptype_dict
-    wfn.nbody_body_energy = energy_body_dict
-    wfn.nbody_body_ptype = ptype_body_dict
+    dicts = [intermediates_dict, energies_dict, ptype_dict, energy_body_dict, ptype_body_dict, nbody_dict]
+    for d in dicts:
+        if d is not None:
```

Comment:
Sure this is `if d is not None:` rather than `if d:`? Those empty objects are `{}`, right, so they evaluate to `False`, but they're not the singleton `None`, I don't think.

Reply:
Other than explicitly casting `str(var)` for variable setting, since `energies_dict` does not use string keys, this seems the proper way to do it.  Good catch, thanks!